packages/frontend/src/lib/dashboard/Dashboard.svelte

```diff
@@ -66,11 +66,11 @@ onMount(async () => {
 });
 
 // Each time bootcAvailableImages updates, check if 'quay.io/bootc-extension/httpd' is in RepoTags
-$: {
+$effect(() => {
   if (bootcAvailableImages?.some(image => image.RepoTags?.includes(exampleImage))) {
     imageExists = true;
   }
-}
+});
```

Comment:
imageExists should be a derived state rather than having an effect updating the imageExists state

Reply:
Good catch, fixed.